### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Updating favicons
- Updating page title
- Adding meta tags
- Fixing margins
- Fixing several styles

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: components/pages/CreateSecret.tsx [hunk 1]
```diff
@@ -18,102 +19,140 @@ export const CreateSecret = (props: Props) => {
...
+          content="Send your secret, securely. One time link to share passwords, keys, and other sensitive data."
+          property="twitter:description"
+        />
+        <meta
+          content="https://cdn.basistheory.com/images/seo/bt-opengraph.png"
+          property="twitter:image"
+        />
+        <meta property="og:type" content="website" />
+        <meta content="summary_large_image" name="twitter:card" />
+      </Head>
```
Comment (dhudec) [inline components/pages/CreateSecret.tsx:55]: None of this head information should be in each component - it should be in _document.tsx

**Review by dhudec — CHANGES_REQUESTED** [review]
The build is broken, and all the head information needs to be moved

File: pages/_app.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ const MyApp = ({
   return (
     <NetworkErrorBoundary Component={ErrorPage}>
       <Head>
-        <title>{'Basis Theory'}</title>
+        <title>{'sendsecurely.ly'}</title>
```
Comment (dhudec) [inline pages/_app.tsx:23]: 👀

File: pages/_document.tsx [hunk 3]
```diff
@@ -47,6 +47,40 @@ class MyDocument extends Document {
...
+          <meta
+            content="SendSecure.ly - Send your secret, securely."
+            property="og:title"
+          />
+          <meta
+            content="One time link to share passwords, keys, and other sensitive data."
+            property="og:description"
+          />
+          <meta
+            content="https://cdn.basistheory.com/images/seo/bt-opengraph.png"
```
Comment (dhudec) [inline pages/_document.tsx:67]: reminder - update these og images

**Review by dhudec — APPROVED** [review]

File: components/pages/CreateSecret.tsx (hunk 1 above)
Comment (jleon15) [reply components/pages/CreateSecret.tsx]: There were a couple page specific tags

**Pull request merged**